addons/mail/static/src/owl/components/activity/activity.scss

```diff
@@ -0,0 +1,174 @@
+// ------------------------------------------------------------------
+// Layout
+// ------------------------------------------------------------------
+
+.o_Activity {
+    display: flex;
+    flex: 0 0 auto;
+    padding: 5px;
+}
+
+.o_Activity_detailsUserAvatar {
+    margin-right: 5px;
+    object-fit: cover;
+    height: 18px;
+    width: 18px;
+}
+
+.o_Activity_icon {
+    position: absolute;
+    top: auto;
+    left: auto;
+    bottom: -5px;
+    right: -5px;
+    width: 25px;
+    height: 25px;
+    object-fit: cover;
+    padding: 6px 5px;
+    text-align: center;
+    line-height: 1.2;
+    border-width: 2px;
```

Comment:
There is the `$border-width` variable but it's 1px, for consistency I don't think we'd want a 2px border here?

Reply:
We need 2px border to get the white circle on icon 
![image](https://user-images.githubusercontent.com/48204996/75665837-161ab700-5c75-11ea-9b4c-5c92cca69c43.png)